src/server/common/model/answer/owner-full-name/owner-full-name.test.js

```diff
@@ -0,0 +1,17 @@
+import { FullNameAnswer } from '../full-name/full-name.js'
+import { OwnerFullNameAnswer } from './owner-full-name.js'
+
+describe('OwnerFullNameAnswer', () => {
+  it('should be a full name', () => {
+    expect(new OwnerFullNameAnswer()).toBeInstanceOf(FullNameAnswer)
+  })
+
+  it('should define the right empty input messages', () => {
+    expect(OwnerFullNameAnswer.config.validation.firstName.empty?.message).toBe(
+      'Enter the first name of the County Parish Holding (CPH) owner'
```

Comment:
I'll raise a separate ticket over this, but this ought to be 'Enter the first name of the registered owner of the cattle'